cheeseshop/dbapi.py

```diff
@@ -433,20 +437,42 @@ def __init__(self, event_id, map_id):
         self.map_id = map_id
 
 
-async def create_schema(conn):
-    await Game.create_schema(conn)
-    await Replay.create_schema(conn)
-    await CsGoHltvEventType.create_schema(conn)
-    await CsGoHltvEvent.create_schema(conn)
-    await CsGoSteamId.create_schema(conn)
-    await CsGoDeathEvent.create_schema(conn)
-    await CsGoStreamer.create_schema(conn)
-    await CsGoGsiEvent.create_schema(conn)
-    await CsGoMap.create_schema(conn)
-    await CsGoEventMapRelation.create_schema(conn)
+class SystemConfig(object):
+    @staticmethod
+    async def create_schema(conn):
+        await conn.execute('''
+            CREATE TABLE system_config(
+                key text UNIQUE NOT NULL,
+                value json
+            )
+        ''')
 
+    @staticmethod
+    async def set(conn, key, value):
+        try:
+            await conn.execute('''
+                INSERT INTO system_config (key, value)
+                VALUES ($1, $2)
+            ''', key, value)
+        except asyncpg.exceptions.UniqueViolationError:
+            await conn.execute('''
+                UPDATE system_config
+                SET value = $2 WHERE key = $1
+            ''', key, value)
+        return SystemConfig(key, value)
+
+    async def get(conn, key):
```

Comment:
Shouldn't this also be a static method?

Reply:
hah, yes... how on earth is https://github.com/ibm-dev-incubator/cheeseshop/pull/55/files#diff-b08e73e3f4c0ede00de725c43853da60R11 working

Reply:
You're still providing the correct number of arguments to the method and not calling it on an instance of SystemConfig. The only difference is that by marking it a staticmethod only one copy exists. That is, SystemConfig().get is SystemConfig.get() will be True if get() is a staticmethod and False otherwise.